deploy/Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM progrium/busybox
-MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com
+FROM alpine:3.2
```

Comment:
Did you get a chance to test this change?

AFAIK our integration tests do not use the docker images as of now.